woocommerce/woocommerce-android
# [Payment Method Improvements] Create the "Mark Order as Complete" Button

Closes: https://github.com/woocommerce/woocommerce-android/issues/11525

Depends on this PR being merged first:
https://github.com/woocommerce/woocommerce-android/pull/11569

### Description
Create the "Mark Order as Complete" Button in the Change Due Calculator screen and make sure to get keyboard focus for Cash Received field.


### Testing instructions

0. Enable OTHER_PAYMENT_METHODS
1. Go to an order that needs to be paid
2. Select "Cash"

Expected: You should see the Change Due screen with the num pad open and Cash Received in focus
